### PR TITLE
Zotero citation update

### PR DESCRIPTION
**Pull request by lpranam**
* Resolves: # 
* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by lpranam** [comment]
known bug: inserting numeric citations in the middle will not renumber all citations, instead numbers are in order of insertion for new citations instead of appearance in the document.

**Comment by lpranam** [comment]
The aforementioned problem should be solved now with the latest commit.

File: browser/src/control/Control.Zotero.js [hunk 1]
```diff
@@ -314,30 +316,92 @@ L.Control.Zotero = L.Control.extend({
...
+
+	handleCitationText: function(citationItems) {
+		//Some citations have style so trying to change just inner most text without affecting other html
+		var citationString = '';
+		var citationNode = $(new DOMParser().parseFromString(citationItems[0].citation, 'text/html').body);
+
+		var that = this;
+		citationItems.forEach(function(item) {
+			var itemHTML = $(new DOMParser().parseFromString(item.citation, 'text/html').body);
+			var innerMostNode = itemHTML.find('*:not(:has("*"))')[0];
```
Comment (eszkadev) [inline browser/src/control/Control.Zotero.js:359]: can we try to not use `jQuery` and instead use `HTMLElement.querySelector` or similar API (one dependency less)

Comment (lpranam) [reply browser/src/control/Control.Zotero.js]: done

**Comment by pedropintosilva** [comment]
this now is failing on cypress macro test on CI side.. re-triggering cypress

**Review by eszkadev — APPROVED** [review]

**Pull request merged**